engine/src/main/scala/cromwell/io/ServiceRouterActor.scala

```diff
@@ -0,0 +1,48 @@
+package cromwell.io
+
+import java.io.IOException
+
+import akka.actor.SupervisorStrategy.{Escalate, Restart}
+import akka.actor.{Actor, ActorInitializationException, OneForOneStrategy, Props}
+import akka.routing.SmallestMailboxPool
```

Comment:
BalancingPool might also be good to consider. I say that w/o having looked to see if you need the semantics from the SMP